<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RECH</class>
 <widget class="QDialog" name="RECH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_rech">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_chercher">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>id</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>nom</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>age</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="rech">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>RECHERCHE</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>69</x>
     <y>79</y>
     <width>251</width>
     <height>181</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTableView" name="tableView_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>241</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
